models/cm1/model_mod.f90

```diff
@@ -1147,7 +1180,7 @@ function observation_on_grid(obs_location, ndims)
 
    endif
 else
-   write(string1,*) 'only grids with periodic x and y grids, or non-periodic '
+   write(string1,*) ' This option is not supoorted '
    write(string2,*) 'boundary conditions supported'
```

Comment:
is the original string1 line wrong now?  the new string1 message (besides the typo) doesn't say *which* option is not supported.  also if that line stays as-is, string2 is now a non-sequitur and should be removed.

Reply:
Right now, no code in the "else" statement should be called because I account for instances where x and y are periodic, x alone is periodic, y alone is periodic, and neither x nor y is periodic.  I included that section as a sanity check just in case a user put in an odd combination of boundary condition values that could not be recognized by DART.

Reply:
makes sense, so how about:

```suggestion
   write(string1,*) ' Unsupported combination of periodic boundary conditions.'
   write(string2,*) 'Contact DART support for more help.'
```

Reply:
That looks great